Automattic/jetpack
# Mailchimp Block: Use the Mailchimp logo for the block icon.

This PR swaps the Mailchimp Block icon from the plain email icon, to the Mailchimp logo.

<img width="327" alt="" src="https://user-images.githubusercontent.com/352291/81138915-0dad6800-8fa7-11ea-9dc8-82e03281a6b3.png">

Fixes #15466.

#### Changes proposed in this Pull Request:
* Mailchimp Block: Use the Mailchimp logo for the block icon.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Enhancement to existing feature.

#### Testing instructions:

* Create a new post.
* Insert a Mailchimp block.
* Confirm that the Mailchimp logo displays everywhere the block icon is used.

#### Proposed changelog entry for your changes:
* Mailchimp Block: Use the Mailchimp logo for the block icon.
